akka-cluster-sharding-typed/src/test/scala/akka/cluster/sharding/typed/scaladsl/ClusterShardingSpec.scala

```diff
@@ -303,19 +303,6 @@ class ClusterShardingSpec
       response.s should include(s"${typeKeyWithEnvelopes.name}-peter")
 
       peterRef ! StopPlz()
-
-      // FIXME #26514: doesn't compile with Scala 2.13.0-M5
-      /*
-      // make sure request with multiple parameters compile
-      Behaviors.setup[TheReply] { ctx =>
-        ctx.ask(aliceRef)(WhoAreYou2(17, _)) {
-          case Success(name) => TheReply(name)
-          case Failure(ex)   => TheReply(ex.getMessage)
-        }
-
-        Behaviors.empty
-      }
-     */
```

Comment:
I think the issue was fixed in the Scala compiler. Instead, you can include the code:
```scala
      // make sure request with multiple parameters compile
      Behaviors.setup[TheReply] { ctx =>
        ctx.ask(peterRef, WhoAreYou2(17, _)) {
          case Success(name) => TheReply(name)
          case Failure(ex)   => TheReply(ex.getMessage)
        }

        Behaviors.empty
      }
```

That works with Scala 2.13 and 3.3.